src/documents/management/commands/document_exporter.py

```diff
@@ -243,15 +285,24 @@ def dump(self, progress_bar_disable=False):
 
             # 3.3. write filenames into manifest
             original_name = base_name
+            if self.use_filename_prefix:
+                original_name = os.path.join("originals", original_name)
```

Comment:
Lets keep to using `Path`s where possible.

Reply:
Try that at first, but ended up trying to store Pathlib in the json then, with an exception
see e03c72a584fd7810257f400729c63505d76f20a3